pkg/report/testdata/linux/report/736

```diff
@@ -0,0 +1,97 @@
+TITLE: BUG: unable to handle kernel paging request in __mod_timer
```

Comment:
__mod_timer is also a very common helper with a 1000 call sites. I think we should ignore it as well.

Reply:
It's called directly from only a handful of them:
https://elixir.bootlin.com/linux/v6.14-rc1/A/ident/__mod_timer

Some syzbot reports:

Here it doesn't matter what exact frame we pick - the previous ones are also timer related
https://syzkaller.appspot.com/bug?extid=bf36934adc7979488192
https://syzkaller.appspot.com/bug?extid=a38f38823275415f1d79
https://syzkaller.appspot.com/bug?extid=83a876aef81c9a485ae8 (?)

Here ignoring __mod_timer would give a better title:
https://syzkaller.appspot.com/bug?extid=50abac586029cf8758e0

Reply:
> It's called directly from only a handful of them:
> https://elixir.bootlin.com/linux/v6.14-rc1/A/ident/__mod_timer

You can see in the reports you referenced, and in the test you added, it's called from _other_ functions all of the kernel, not from these few places.

> Here it doesn't matter what exact frame we pick - the previous ones are also timer related

In these we probably need to ignore more functions.

Reply:
Added `__mod_timer` to ignore lists. PTAL.